test/e2e/framework/service_util.go

```diff
@@ -451,6 +462,7 @@ func (j *ServiceTestJig) WaitForEndpointOnNode(namespace, serviceName, nodeName
 	ExpectNoError(err)
 }
 
+// SanityCheckService performs sanity checks of the given services.
```

Comment:
```suggestion
// SanityCheckService performs sanity checks on the given service
```

Reply:
side note: SmokeTest, HealthCheck, Verify would all be better verbs for this method